IOS/SwiftSeedProject/SwiftSeedProject/Domain/Services/PersonService.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PersonService.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/28/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+open class PersonService {
+    let dataProvider : InMemoryDataProvider
+    
+    var persons = [Person]()
+    
+    init() {
+        dataProvider = InMemoryDataProvider()
+        let result : [AnyObject] = dataProvider.getEntities()
```

Comment:
`result` name is waaaaaaaaaaaaay too generic.

Name should be something like `personEntities` or similar.